docs/local.md

```diff
@@ -31,97 +42,145 @@ make build
 the file ..., or if you have `jq` installed (available via `sudo apt install jq`), you can set a
 variable like this:
 ```bash
-CONTRACT_ADDR=$(jq -r .ContractAddress /tmp/xmrtaker/contract-address.json)
+CONTRACT_ADDR=$(jq -r .ContractAddress /tmp/xmrtaker-*/contract-address.json)
 ```
 
-Now start Bob's swapd instance in terminal 2:
+Now start Bob's swapd instance:
 ```bash
-./swapd --dev-xmrmaker --bootnodes "${BOOT_NODE}" --contract-address "${CONTRACT_ADDR}"
+./bin/swapd --dev-xmrmaker --bootnodes "${BOOT_NODE}" --contract-address "${CONTRACT_ADDR}" &> bob.log &
 ```
 
+### Using swapcli To Check Balances
+
+`swapcli` is an executable to interact with a `swapd` instance via it's RPC port on the
+local host (`127.0.0.1`). The current security model of swapd assumes connections
+originating from the local host are authorized, so you should not run `swapd` for
+production swaps on multi-user hosts or hosts with malicious software running on them.
+
 Note: when using the `--dev-xmrtaker` and `--dev-xmrmaker` flags, Alice's RPC server runs
-on http://localhost:5001 and Bob's runs on http://localhost:5002 by default.
+on http://localhost:5000 (the default port) and Bob's runs on http://localhost:5001. Since
+Bob's `swapd` RPC port is not the default, you will need to pass `--swapd-port 5001` to
+`swapcli` when interacting with his daemon.
 
-Now, in terminal 3, we will interact with the swap daemon using `swapcli`.
+Alice and Bob are both using Ethereum wallet keys that are prefunded by Ganache.
+Background Monero Monero mining was started for Bob, because we used the `--dev-xmrmaker`
+flag.
 
-First we need mine some XMR for Bob. Alice already has ETH, because she is using
-a prefunded by ganache address. You can see the balances for Bob with the following
-command:
+You can check Alice's balance with this command:
 ```bash
-./swapcli balances --swapd-port 5002
+./bin/swapcli balances
 ```
-Note that Alice is on the default swapd port of 5001, so the `--swapd-port` flag is optional
-when interacting with her daemon.
-
-To mine some monero blocks for Bob, you can use our bash shell function shown below:
+And Bob's balances with this command:
 ```bash
-source scripts/testlib.sh
-mine-monero-for-swapd 5002
+./bin/swapcli balances --swapd-port 5001
 ```
-Now you can use the second to last command to see Bob's updated monero balance.
+
+### Make a Swap Offer
 
 Next we need Bob to make an offer and advertise it, so that Alice can take it:
 ```bash
-./swapcli make --min-amount 0.1 --max-amount 1 --exchange-rate 0.05 --swapd-port 5002
-# Published offer with ID cf4bf01a0775a0d13fa41b14516e4b89034300707a1754e0d99b65f6cb6fffb9
+./bin/swapcli make --min-amount 0.1 --max-amount 1 --exchange-rate 0.05 --swapd-port 5001
+```
+Example output:
+```
+Published:
+	Offer ID:  0x09dd41c7b8620cdc3716463dc947a11edf3af45ff07c8b0ff89dd23592e732ca
+	Peer ID:   12D3KooWK7989g6xmAaEsKFPuZTj2CVknRxQuk7dFL55CC1rpEWW
+	Taker Min: 0.005000000000000001 ETH
```

Comment:
I'll fix this to what it will look like after we switch to using big.Float and using a precision that is customized for either Monero or Ethereum.